src/pages/Accounts/AccountDetailsPage.vue

```diff
@@ -0,0 +1,606 @@
+<script setup lang="ts">
+import { useRouter } from 'vue-router';
+import { computed, onMounted, ref } from 'vue';
+import { http } from '@/shared/api';
+import BaseContentBlock from '@/shared/ui/components/BaseContentBlock.vue';
+import BaseTabs from '@/shared/ui/components/BaseTabs.vue';
+import DataField from '@/shared/ui/components/DataField.vue';
+import { useTable } from '@/shared/lib/table';
+import { formatMoney, numberFormatter } from '@/shared/utils/money-formatters';
+import BaseTable from '@/shared/ui/components/BaseTable.vue';
+import type { TableAsset } from './types';
+import { getFakeAssets } from './utils';
+import BaseLink from '@/shared/ui/components/BaseLink.vue';
+import {
+  type filterTransactionsModel as ftm,
+  TransactionStatusFilter,
+  TransactionTypeFilter,
+} from '@/features/filter-transactions';
+import BaseHash from '@/shared/ui/components/BaseHash.vue';
+import TransactionStatus from '@/entities/transaction/TransactionStatus.vue';
+import { transactionModel } from '@/entities/transaction';
+import { computedEager, useWindowSize } from '@vueuse/core';
+import { format } from '@/shared/lib/time';
+import { adaptiveTransactionTypeOptions, assetsItems } from './consts';
+import BaseLoading from '@/shared/ui/components/BaseLoading.vue';
+import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
+
+const router = useRouter();
+
+const { handleUnknownError } = useErrorHandlers();
+
+const HASH_BREAKPOINT = 1200;
+const { width } = useWindowSize();
+
+const hashType = computedEager(() => (width.value < HASH_BREAKPOINT ? 'short' : 'medium'));
```

Comment:
You mentioned yourself that `computedEager` isn't needed in Vue 3.3+

But I see that you haven't updated Vue itself. Can you please bump its version too?